### PR TITLE
chore: upgrade front-end dependencies

### PR DESCRIPTION
**Pull request by hmatalonga**
This PR upgrades the front-end dependencies and also replaces yarn with npm as default package manager.

Some styling changes will be added in order to migrate some UI library components.

### CODE REVIEW

**Comment by fntneves** [comment]
Is this WIP, yet?

**Comment by hmatalonga** [comment]
Yes, I will add a few more tweaks... I will request a review once is done :+1:

**Comment by fntneves** [comment]
@hmatalonga What about this one?

**Comment by hmatalonga** [comment]
In this particular one, it will fix the CI build but it will potentially introduce UI breaking changes which can be fix before or after merging any of the another pull requests...

**Comment by fntneves** [comment]
@hmatalonga Can you please fix UI for these updated dependencies?

**Comment by djcouto** [comment]
Hope this is not out of scope, but I would suggest changing this PR to update front-end and backend dependencies, because I just installed the project locally and I get this warnings:
```
Package phpoffice/phpexcel is abandoned, you should avoid using it. Use phpoffice/phpspreadsheet instead.
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
Package satooshi/php-coveralls is abandoned, you should avoid using it. Use php-coveralls/php-coveralls instead.
```

If it's out of scope please let me know so I can create a new PR with the changes.

**Comment by fntneves** [comment]
@djcouto Since the next PRs depend on this one, I suggest to finish this first and then focus on backend, which is not critical for the proposed features.

**Comment by djcouto** [comment]
Due to inactivity, I will take control of this PR. 
@fntneves the front-end dependencies upgraded and I check every single view and everything is looking good now.

Whenever you can give me some feedback so this can be merged asp 👍

File: resources/assets/sass/app.scss [hunk 1]
```diff
@@ -8,12 +8,16 @@
 // Bootstrap
 @import "node_modules/bootstrap/scss/bootstrap";
 
+/* Vue Select */
+// @import "node_modules/vue-select/dist/vue-select";
```
Comment (fntneves) [inline resources/assets/sass/app.scss]: Should we remove this?

File: package.json [hunk 2]
```diff
@@ -3,26 +3,30 @@
...
-    "bootstrap": "^4.0.0-beta",
-    "bootstrap-vue": "^1.0.0-beta",
-    "cross-env": "^3.2.3",
-    "element-ui": "^1.4.2",
-    "jquery": "^3.1.1",
-    "laravel-mix": "^1.0",
-    "lodash": "^4.17.4",
+    "axios": "^0.18.1",
+    "bootstrap": "^4.3.1",
+    "bootstrap-vue": "^2.0.0-rc.11",
```
Comment (fntneves) [inline package.json]: Maybe upgrade this? It is currently on -rc28.

**Comment by djcouto** [comment]
`npm audit` returns 0 vulnerabilities, so I think we can safely say that we don't need to worry about dependency-related security warnings 👍

**Comment by fntneves** [comment]
You're the best @djcouto  <3

**Pull request merged**